src/components/Form.js

```diff
@@ -75,12 +75,18 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
-    const mustValidate = !this.props.noValidate && (this.props.liveValidate || options.validate);
+  onChange = (formData, newErrorSchema) => {
```

Comment:
Base on my understanding so far, the "option.validate" was just to notify Form component to reorder or remove errorSchema in the array as well by using "validation" when the array was updated. If it wasn't the exiting "option" parameter that can be replaced with, I would probably not propose this solution for the moment.